src/jshint.js

```diff
@@ -5071,12 +4993,16 @@ var JSHINT = (function() {
       },
 
       current: {
+        getlabel: function(t) {
```

Comment:
I like seeing more code organized into the "blockScope" object, but I think this method might be too powerful. As defined, the previously-internal "label" objects are exposed, and we'll have to be vigilant about how future patches handle those. For your purposes here, though, you don't need access to the whole object, and you don't need to modify it, either. A more special-purpose method would be safer and would also reduce the repetition in the places where you use it--something like `labelType(t)` that returns the string value (or `null` when not present).